src/stylesheets/core/_properties.scss

```diff
@@ -501,7 +501,9 @@ $system-properties: (
     standard: map-collect(
       map-get($system-spacing, 'smaller'),
       map-get($partial-values, 'zero-zero'),
-      ('05': spacing-multiple(0.5))
+      (
+        '05': spacing-multiple(0.5),
```

Comment:
wrote this in expanded format to better accept new custom values if we decide to accept, say, `1` unit